src/components/FilterTag.svelte

```diff
@@ -2,6 +2,7 @@
 import { createEventDispatcher } from 'svelte'
 
 export let label
+export let showCloseButton = true
```

Comment:
I'm a little confused why this one is in here, was this from some of Matt's work?

Reply:
No, it's so I can use `FilterTag` in two different places. Have an alternate solution?

Reply:
If you can find a name where a default of `false` makes sense (e.g. `hideCloseButton`), that has the benefit of letting us simply include the attribute (no value) when we want to hide the close button, and simply not use the attribute when we want the default.

Example with close button:
```svelte
<FilterTag label="Keyword: Altoids" on:remove={onRemove} />
```
Example without close button:
```svelte
<FilterTag label="Keyword: Altoids" on:remove={onRemove} hideCloseButton />
```